code_annotations/base.py

```diff
@@ -578,66 +578,33 @@ def _format_results_for_report(self, all_results):
         Returns:
             Dict of results arranged for reporting
         """
-        group_children = self._get_group_children()
         formatted_results = {}
         current_group_id = 0
-
         for filename in all_results:
-            self.echo.echo_vv(f"report_format: formatting {filename}")
```

Comment:
Curious why you wanted to drop this top-level echo.

Reply:
I shouldn't have -- I'll add it back.